pkg/securitycontextconstraints/sccadmission/admission.go

```diff
@@ -217,6 +217,14 @@ loop:
 	for i, provider = range providers {
 		if !allowedForUserOrSA(provider) {
 			denied = append(denied, provider.GetSCCName())
+			// this will cause every security context constraint attempted, in order, to the failure
+			validationErrs = append(validationErrs,
+				field.Forbidden(
+					field.NewPath(fmt.Sprintf("provider %q: ", provider.GetSCCName())),
```

Comment:
that's not a path.

How would this end up in logs?

Reply:
this eventually looks like 'provider "tehprovider": :  Forbidden: erro' <-- please see to those `: :`